.github/workflows/master.yml

```diff
@@ -144,6 +144,7 @@ jobs:
     - uses: ./.github/actions/setup-python
       with:
         python-version: "3.7"
+    - uses: ./.github/actions/setup-pyenv
```

Comment:
Added `setup-pyenv` here to run it and make sure it works. In https://github.com/mlflow/mlflow/pull/5380, we might need to add this action in other jobs.